setup.py

```diff
@@ -74,15 +77,17 @@ def run(self):
 with open('README.rst') as readme:
     long_description = readme.read()
 
-
+extras_require = None
+data_files = None
 if __name__ == '__main__':
     if platform.system() != 'Windows':
         data_files = [('man/man1', ['coala.1'])]
     else:
         data_files = None
 
+if __name__ == '__main__':
     setup(name='coala',
-          version=VERSION,
+          version='0.12.0.dev99999999999999',
```

Comment:
I'm confused over why to hardcode the version when [Line 68](https://github.com/coala/coala/pull/4897/files#diff-2eeaed663bd0d25b7e608891384b7298R68) will get us the same version.

Reply:
hmm.  actually no, that only happens when 'dev' is in the version.
but, line 68 is definitely problematic; will need to think through how that should work in a moban controlled system.

Reply:
I have a suggestion don't know if it would be good to do so.

can we always adjust the version (using `.misc/adjust_version_number.py`) rather than adjusting only when there is a dev, that way maybe we won't need to hardcode it.

Reply:
@Monal5031 This is controlled by the template. It doesn't matter since it'll be generated. If the file isn't updated, CI will fail anyway.